app/controllers/admin/document_sources_controller.rb

```diff
@@ -1,29 +0,0 @@
-class Admin::DocumentSourcesController < Admin::BaseController
-  before_action :require_import_permission!
```

Comment:
It looks like we can now remove the method `require_import_permission!` from Base Controller:

https://github.com/alphagov/whitehall/blob/fc1d1208c2494257ad33e5f1563c30b2bf9ea5b4/app/controllers/admin/base_controller.rb#L21-L23

As well as the "import" permission in the User model:

https://github.com/alphagov/whitehall/blob/ba7e7191b0b5230eb59e3cddb22ad79386b76cb6/app/models/user.rb#L23

https://github.com/alphagov/whitehall/blob/ba7e7191b0b5230eb59e3cddb22ad79386b76cb6/app/models/user.rb#L83-L85

These seem to have only been used for the Document Sources functionality.